formserver/Dockerfile

```diff
@@ -4,4 +4,22 @@ ARG DEPENDENCY=target/dependency
 COPY ${DEPENDENCY}/BOOT-INF/lib /app/lib
 COPY ${DEPENDENCY}/META-INF /app/META-INF
 COPY ${DEPENDENCY}/BOOT-INF/classes /app
+
+RUN \
+    apt-get update && \
+    apt-get -y upgrade && \
```

Comment:
I'm not sure about putting in an `upgrade` here -- doesn't this open us up to "somebody pushed a minor version with a bugfix and now our stuff is broken"? idk how likely it is but I don't see the point.

Reply:
Yea, good point. I'll test it again without and make sure it works.